testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/cluster/AuthenticationSessionClusterTest.java

```diff
@@ -99,11 +99,21 @@ public void testAuthSessionCookieWithAttachedRoute() {
             String route = authSessionCookie.substring(authSessionCookie.indexOf(".") + 1);
             visitedRoutes.add(route);
 
+            getTestingClientFor(backendNode(0)).server().run(session -> {
```

Comment:
`TopologyInfo#getRouteName` is still being used on L44 in this class. Should this now be using `StickySessionEncoderProvider#SessionIdAndRoute` instead? If so, should we deprecate the `getRouteName` method?

Reply:
> I can change the test to assert that the route is null if you agree.

Yes I think that would be better.

> I would love to deprecate `TopologyInfo` and remove it.

+1 it would be nice to remove usage and deprecate it, lets create a separate issue for that.

I see that `amIOwner` is currently never used, so in this PR we should replace the usage of `getRouteName` in the test and deprecate the unused public methods.